src/spanish-tortilla.md

```diff
@@ -0,0 +1,38 @@
+# Spanish Tortilla
+
+![Spanish Tortilla](pix/spanish-tortilla.webp)
+
+- ⏲️ Prep time: 5 min
+- 🍳 Cook time: 35 min
+- 🍽️ Servings: 6
+
+## Ingredients 
+
+* 8 free-range eggs
+* 1 kg of potatoes
+* 1 white onion
+* Extra virgin olive oil
+
+## Instructions 
+
+1. Peel, rinse and dry the potatoes.
+2. Cut the potatoes into thin slices. Put in a bowl, add salt to taste and stir.
+3. Put a nonstick frying pan on the fire and add enough olive oil to fry all the potato slices.
+4. Add the potato slices already salted and let them cook for approximately 20 minutes over low heat.
+5. While the potatoes are frying, beat the eggs in a bowl. Peel and cut the onion as finely as possible.
+6. Add olive oil in another frying pan and slowly fry the onion until it gets a slightly golden color. 
+7. Drain the onion and add it to the bowl with the beaten egg.
+8. Remove the potatoes from the pan with a slotted spoon, draining the oil away as much as possible, and add them to the bowl with the onion and egg.
+9. Let the mixture rest for at least 5 minutes so all the flavors come together. Give it a very gentle stir from time to time.
+10. Remove the oil from the pan you used to fry the potatoes. Use the pan to cook the mixture over medium-high fire for about 5 minutes.
+11. Flip it over with the help of a big plate and cook the other side for another 5 minutes.
+
+## Notes
+
+* Things like the slicing of the potato, the amount of salt and how curdled is the inner part of the tortilla depend on your own taste. Don't be afraid to experiment!
+
+## Contribution
+
+OctopusOnFire - [Minds] (https://www.minds.com/OctopusOnFire/)
```

Comment:
```suggestion
OctopusOnFire - [Minds](https://www.minds.com/OctopusOnFire/)
```

Markdown will make a link for you if you get rid of the space between the `[...]` and `(...)`